### DISCUSSION TITLE
Hosting a fork of monkeytype with GitHub Pages for testing purposes

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1436] [Questions] [answered]
I was wondering how one would be able to host a fork of Monkeytype of GitHub pages. @SeerLite's fork is a [good example](https://github.com/SeerLite/monkeytype) of what I'm trying to accomplish https://seerlite.github.io/monkeytype/. The problem is Pages displays the `README.md` instead of the `index.html` file. I believe the `README.md` is shown because the `index.html` is in a sub-folder. Does anyone know how this could be resolved? Here is what I have at the moment https://unrealapex.github.io/monkeytype/.

**Comment by SeerLite** [comment, marked as answer]
Hahaha I didn't expect anyone to really check my fork out. It's just an experiment I did to see if I could get it to function without all the firebase and analytics stuff. Check out the [degoogle](https://github.com/SeerLite/monkeytype/tree/degoogle) branch and it's package.json. I haven't merged master to it yet because I've been more focused on the input rewrite PR.

At first my intention was to publish a fork for people who wanted to avoid Google for privacy reasons, but I never asked Miodec about degoogling the main site in the first place so I thought it wouldn't be fair. Then I got lazy and just forgot about it. Also I don't trust myself to maintain it for long.

My main use for it now is just to be able to develop offline. Where I live the internet isn't always stable so I can't get `firebase serve` to work all the time. Though I do have to do some constant cherry-picking and merging for PRs, and I have to pay attention to how it all comes together so I don't mess up the history ::
EDIT: Oh and obviously I've also been using it to set up the preview pages you've probably seen
